server/polar/models/organization.py

```diff
@@ -111,6 +112,29 @@ class OrganizationCustomerEmailSettings(TypedDict):
 }
 
 
+class CustomerPortalUsageSettings(TypedDict, total=False):
```

Comment:
I think we can drop `total=False` on each schema: with the migration, all existing orgs will have the settings filled.